From: Free Ekanayaka <freee@debian.org>
Date: Fri, 17 Feb 2017 11:43:16 +0000
Subject: dont-depend-on-m2r

We don't need to generate the README at setup time, so disable
the dependency on m2r.
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 5971d97..c0b221e 100644
--- a/setup.py
+++ b/setup.py
@@ -8,7 +8,7 @@ try:
     from m2r import parse_from_file
     long_description = parse_from_file('README.md')
 except(IOError, ImportError):
-    print("\n\n!!! m2r not found, long_description is bad, don't upload this to PyPI !!!\n\n")
+    #print("\n\n!!! m2r not found, long_description is bad, don't upload this to PyPI !!!\n\n")
     import io
     long_description = io.open('README.md', encoding="utf-8").read()
 
@@ -24,7 +24,7 @@ setup(
     package_dir={'automat': 'automat'},
     setup_requires=[
         'setuptools-scm',
-        'm2r',
+#        'm2r',
     ],
     install_requires=[
         "attrs",
